TROUBLESHOOTING.md

```diff
@@ -10,4 +10,7 @@ To troubleshoot issues with Imageset configuration generation, the `oc-mirror li
 When submitting bug reports, include the metadata.json from the problem generated tar artifact, called an imageset. The metadata.json can be retrieved from the imageset with: `tar xf <imageset-name> /publish/.metadata.json`. Then retrieve and upload the metadata.json from  `./publish/.metadata.json`. 
 
 ## Operator Installation
-To troubleshoot issues with the created file-based catalog, use the following command to get information about the problem package, ‘oc get packagemanifests -n openshift-marketplace <packagename> -o json | jq '.status.channels[]|{name: .name, currentCSV: .currentCSV}'
\ No newline at end of file
+To troubleshoot issues with the created file-based catalog, use the following command to get information about the problem package, ‘oc get packagemanifests -n openshift-marketplace <packagename> -o json | jq '.status.channels[]|{name: .name, currentCSV: .currentCSV}'
+
+## Docker Registry parsing
```

Comment:
Nit: I would make this title something more generic. Maybe `Destination Registry Parsing`

Reply:
I can do that. I chose Docker registry parsing because this only happens for `docker://` registry destinations, and I think "docker registry parsing" is somewhat close to what a confused user might search for if they don't read the warning output